engine-tests/src/test/java/org/terasology/engine/integrationenvironment/MTEExtensionTestWithPerMethodLifecycle.java

```diff
@@ -64,33 +60,31 @@ public static void createEntity(EntityManager entityManager, TestInfo testInfo)
     @Order(1)
     public void firstTestFindsThings() {
         List<EntityRef> entities = Lists.newArrayList(entityManager.getEntitiesWith(DummyComponent.class));
-        // There should be one entity, created by the @BeforeAll method
+        // There should be one entity, created by the @BeforeEach method
         assertEquals(1, entities.size());
 
         DummyComponent component = entities.get(0).getComponent(DummyComponent.class);
-        assertTrue(component.eventReceived);
+        assertThat(component.eventReceived).isTrue();
+        assertThat(seenNames).isEmpty();
 
         // Remember that a test has seen this one.
-        assertNotNull(component.name);
-        assertFalse(seenNames.containsKey(component.name));
-        seenNames.put(component.name, 1);
+        assertThat(component.name).isNotNull();
+        assertThat(seenNames).isEmpty();
+        seenNames.add(component.name);
     }
 
     @Test
     @Order(2)
-    public void thingsStillExistForSecondTest() {
+    public void thingsDoNotPolluteSecondTest() {
         List<EntityRef> entities = Lists.newArrayList(entityManager.getEntitiesWith(DummyComponent.class));
-        // There should be one entity, created by the @BeforeAll method
+        // There should be one entity, created by the @BeforeEach method for this one test
         assertEquals(1, entities.size());
 
-        // Make sure that this is the same one that the first test saw.
+        // FIXME: what to assert here does this make sense or should we drop the whole thing
+        //     and say it's covered by LifecyclePerMethodInjectionTest?
```

Comment:
I'm not really sure about this either - I definitely want to keep the part of the test chat checks for which entities/how many are registered. Not sure about the part below, but maybe we can keep it for now with this comment to feel less bad when eventually removing it ^^